src/Files.App/ViewModels/SettingsViewModels/PreferencesViewModel.cs

```diff
@@ -475,6 +491,87 @@ public bool ListAndSortDirectoriesAlongsideFiles
 			}
 		}
 
+		private int selectedDefaultSortingIndex;
+		public int SelectedDefaultSortingIndex
+		{
+			get => selectedDefaultSortingIndex;
+			set
+			{
+				if (SetProperty(ref selectedDefaultSortingIndex, value))
+				{
+					OnPropertyChanged(nameof(SelectedDefaultSortingIndex));
+					DefaultSortOption = value < 5 ? (SortOption)value : SortOption.FileTag;
```

Comment:
Same as above about the magic constant. If the `5` here has the same use as line 115, putting it inside a `private readonly int` field could solve the issue quite nicely.